villain/interp.rkt

```diff
@@ -123,7 +135,7 @@
 
 ;; Defns Symbol -> Defn
 (define (defns-lookup ds f)
-  (findf (match-lambda [(Defn g _ _) (eq? f g)])
+  (findf (match-lambda [(Defn g _ _) (equal? f g)])
```

Comment:
This seems like a search/replace bug.  It should remain `eq?` unless I'm missing something.